src/Subscriber/CloseDraftPRSubscriber.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Api\Issue\IssueApi;
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use App\Service\ComplementGenerator;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CloseDraftPRSubscriber implements EventSubscriberInterface
+{
+    private $issueApi;
+    private $complementGenerator;
+
+    public function __construct(IssueApi $issueApi, ComplementGenerator $complementGenerator)
+    {
+        $this->issueApi = $issueApi;
+        $this->complementGenerator = $complementGenerator;
+    }
+
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        $repository = $event->getRepository();
+        if ('opened' !== $data['action'] || !($data['pull_request']['draft'] ?? false)) {
+            return;
+        }
+
+        $number = $data['pull_request']['number'];
+        $complement = $this->complementGenerator->getPullRequestComplement();
+        $this->issueApi->commentOnIssue($repository, $number, <<<TXT
+Hey!
+
+$complement
+
+Could you please click the "ready for review" button? Or maybe close this PR and open a new one when you are done.
+Note that a pull request does not have to be "perfect" or "ready for merge" when you first open it, but it should be ready for a first review.
+
+Cheers!
+
+Carsonbot
+TXT
+);
+
+        // TODO add a scheduled task to close the PR within 1 hour.
```

Comment:
This will be implemented after #102 

It is not a blocker for merging this PR.